### PR TITLE
GitHub OAuth

### PR DESCRIPTION
**Pull request by manics**
Adds an optional script and instructions for setting up GitHub OAuth on devspace.

### CODE REVIEW

File: home/init.groovy.d/github-oauth.groovy.disabled [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+// GitHub OAuth scopes
+String oauthScopes = 'read:org'
+// List of GitHub users who are Jenkins admins
+String adminUserNames = 'microscopepony,snoopycrimecop'
+// Allow access to members of this GitHub organisation
+String organizationNames = 'ome'
+
+// Permissions: you probably don't need to change these
+
+// Use Github repository permissions
```
Comment (mtbc) [inline home/init.groovy.d/github-oauth.groovy.disabled]: GitHub

**Comment by joshmoore** [comment]
Successfully installed on the (temporarily running) east-ci if anyone would like to try it out. I imagine a few of the properties will want adjusting under the "no changes" section.

**Comment by manics** [comment]
Updated

**Comment by joshmoore** [comment]
Thanks, @manics. Since the config file needs manual editing anyway, merging for now. But looks like a solid solution.

**Pull request merged**